### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by sumitk-singh**

### CODE REVIEW

File: pages-css/compo-css/navbar.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
+        /* simple nav-bar */
+        
+        .nav-bar {
+            display: flex;
+            flex-direction: row;
```
Comment (iRohitGaur) [inline pages-css/compo-css/navbar.css:5]: direction is row by default. Any reason to put this?

File: pages-css/home-page-css/page-layout-handeler.css [hunk 2]
```diff
@@ -0,0 +1,137 @@
+:root {
+    --theam-background: rgb(206, 206, 206);
+    --theam-color: rgb(252, 1, 1);
+    --theam-border-radius: 15px;
+}
+
+* {
+    margin: 0px;
+    padding: 0px;
```
Comment (iRohitGaur) [inline pages-css/home-page-css/page-layout-handeler.css:9]: only 0 is fine. No need for 0px.

**Review by iRohitGaur — APPROVED** [review]
Nice work! Clean and understandable code. Good to merge with changes

File: pages-css/compo-css/navbar.css (hunk 1 above)
Comment (sumitk-singh) [reply pages-css/compo-css/navbar.css]: ok, i get it.

File: pages-css/home-page-css/page-layout-handeler.css (hunk 2 above)
Comment (sumitk-singh) [reply pages-css/home-page-css/page-layout-handeler.css]: done

**Pull request merged**